nemo/collections/nlp/metrics/sequence_perplexity.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from pytorch_lightning.metrics import Metric
```

Comment:
PyTorch lightning metrics are deprecated in favor of TorchMetrics (https://torchmetrics.readthedocs.io/en/latest/). Do you think it's possible to easily adapt this to use TorchMetrics instead?

Reply:
It has pretty similar semantics - https://torchmetrics.readthedocs.io/en/latest/pages/implement.html